flutter/lib/src/default_integrations.dart

```diff
@@ -444,6 +410,30 @@ class _LoadAndroidImageListIntegrationEventProcessor extends EventProcessor {
 
     return event;
   }
+
+  static bool needsSymbolication(SentryEvent event) {
+    final frames = event.exceptions?.first.stackTrace?.frames;
+    if (frames == null) return false;
+    return Platform.isAndroid
+        ? frames.any((frame) => 'native' == frame.platform)
+        : frames.isNotEmpty;
+  }
+
+  static SentryEvent copyWithDebugImages(
```

Comment:
If we reuse the `loadImageList` method channel call, we don't need to make this static and non-private, but rather being used only within the `_LoadImageListIntegrationEventProcessor` integration.